src/main/webapp/WEB-INF/web.xml

```diff
@@ -17,10 +17,10 @@
   specific language governing permissions and limitations
   under the License.
 -->
-<web-app version="3.1"
-         xmlns="http://xmlns.jcp.org/xml/ns/javaee"
+<web-app version="6.0"
```

Comment:
This requires servlet API 6, maybe its better to use 5?

Reply:
Updated it to use 5, thanks